covid19_npis/model/number_of_tests.py

```diff
@@ -3,6 +3,7 @@
 import logging
 import pymc4 as pm
 import numpy as np
+import pandas as pd
```

Comment:
We should not need pandas in this, submodule. Convertion to tensors should be done in the modelParams.

Reply:
resolved - I didn't find any function/variable in the dataframe that already covered this and added an according function "get_weekdays" to the modelParams class